### ISSUE TITLE
reproduce performace on occluded re-id

### ISSUE THREAD

**Issue by InsAItTeam** [CASIA-IVA-Lab/ISP-reID#7]
Thanks for the amazing implementations.
Some questions regarding reproducing the reported performances. I've been able to obtain really close performances on Market and Duke, however on Occluded Duke, using defalut bash file, I only got around 50.4/58.8(mAP/Rank-1) which is lower than reported 52.3/62.8. I also tried to increase the number of clusters, the performance was droped.
Can you shed some insights on how to improve the performance? Many thanks!

Also why there are two evaluations (with and without arm)?

**Comment by zhukuan96** [comment]
I am not sure why you get lower performance, but I think you can try: (1) Using one triplet: Changing the three triplet losses in layers/__init__.py to 'triplet(torch.cat((y_global, y_fore, y_part), dim=1), cls_target)[0]'. (This is most likely to succeed.) (2) Reducing the number of clusters to 4. (3) Open the center_loss, open the clustering.enhanced.

In the traning phase, the triplet loss is calculated without using the arm, so the arm is not so effective on holistic person reid datasets. We are solving this problem in our journal version.

**Comment by InsAItTeam** [comment]
Thanks.

**Comment by GJTNB** [comment]
@JasAva My hardware condition is not very good. Can you provide the model you trained.

**Comment by PierreDim** [comment]
Did you manage to reproduce the paper results on Occluded Duke? If yes, which settings did you use? Thank you
